ai_chatbots/tools.py

```diff
@@ -212,23 +220,26 @@ class VideoGPTToolSchema(pydantic.BaseModel):
 
 
 @tool(args_schema=SearchContentFilesToolSchema)
-def search_content_files(q: str, state: Annotated[dict, InjectedState]) -> str:
+def search_content_files(
+    q: str, state: Annotated[dict, InjectedState], readable_id: str | None = None
```

Comment:
I thought possibly having this exposed as a param vs being an injected state would make this prone to hallucination but it seems to be doing well (only time i could get it to do some thing wrong was if i explicitely told it to tell me about "readable_id=test"